service.kn.switchtimer/addon.xml

```diff
@@ -1,23 +1,23 @@
 <?xml version='1.0' encoding='UTF-8' standalone='yes'?>
-<addon id="service.kn.switchtimer" name="KN Switchtimer Service" provider-name="Birger Jesch" version="1.2.4">
+<addon id="service.kn.switchtimer" name="KN Switchtimer Service" provider-name="Birger Jesch" version="2.0.1">
     <requires>
-        <import addon="xbmc.addon" version="16.0.0"/>
-        <import addon="xbmc.python" version="2.24.0"/>
-        <import addon="xbmc.json" version="6.32.4"/>
+        <import addon="xbmc.addon" version="14.0.0"/>
+        <import addon="xbmc.python" version="2.20.0"/>
```

Comment:
2.24.0 was correct for `xbmc.python` since this is in Jarvis branch. Why did you change it for the Isengard value?